### PR TITLE
ci: add go 1.17 to test matrix

### PR DESCRIPTION
**Pull request by anajavi**
Adds go 1.17 to github actions test matrix.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x, 1.16.x]
+        go-version: [1.14.x, 1.15.x, 1.16.x, 1.17.x]
```
Comment (dgsb) [inline .github/workflows/ci.yaml:50]: We can remove all former version and only keep 1.17.x in order to speed up CI runs.

**Comment by dgsb** [comment]
Thanks @anajavi, can update your PR according to my review ?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (anajavi) [reply .github/workflows/ci.yaml]: Done.

**Comment by anajavi** [comment]
> Thanks @anajavi, can update your PR according to my review ?

I've updated the PR.

**Pull request merged**